### PR TITLE
Add the 2201.10.0 release note for supporting package resources

### PR DESCRIPTION
**Pull request by Dilhasha**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -111,6 +111,42 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### Improvements
 
+- With this update, resources have been moved to the package level and module-level resources are deprecated. Resources available in module resource paths will be ignored. To be recognized as valid, any existing module-level resources must be relocated to the resources directory in the package root.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:114]:
```suggestion
- Resources are now expected at the package level, and module-level resources are no longer supported. Resources that were previously included at module-level, have to be moved from modules to the package root to continue to be identified as resources.
```

Is deprecated correct? May imply it's still supported?

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -111,6 +111,42 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+  Any resources within the current package, as well as those exported from package dependencies, can now be accessed through an interop function.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:115]: Do we need this? Should the user write this? If so, not sure if we should emphasize it in the release note.

Comment (Dilhasha) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md]: Added this as a note with 2bcdfaa

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -111,6 +111,43 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ### Improvements
 
+- Resources are now expected at the package level, and module-level resources are no longer supported. Resources that were previously included at module-level, have to be moved from modules to the package root to continue to be identified as resources.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:114]:
```suggestion
- Resources are now expected at the package level, and module-level resources are no longer supported. Resources that were previously included at module-level have to be moved from modules to the package root to continue to be identified as resources.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -111,6 +111,43 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+
+  >**Note:** Any resources within the current package, as well as those exported from package dependencies, can be accessed through an interop function.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:116]:
```suggestion
  >**Note:** Any resources within the current package, as well as those exported from package dependencies, can be accessed via an [external function](https://ballerina.io/learn/by-example/interface-to-external-code/).
```

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**